src/Common/Allocator.cpp

```diff
@@ -57,7 +57,7 @@ void prefaultPages([[maybe_unused]] void * buf_, [[maybe_unused]] size_t len_)
     auto [buf, len] = adjustToPageSize(buf_, len_, page_size);
     if (::madvise(buf, len, MADV_POPULATE_WRITE) < 0)
```

Comment:
> The best option would be to call madvise only if the kernel is recent enough but a) the path is performance-critical and b) as time goes on, the error will happen decreasingly often (as < 5.14 is less and less used).

Why not add the same code as we have for rename2? This should not add significant overhead that will be noticable for madvise

> Therefore lowered the rate at which the error is logged from 1 to 60 sec.

Or do we even need multiple messages of this error, one error is enough to me

Reply:
Good idea. Changed to be like in rename2!